composer.json

```diff
@@ -203,5 +203,6 @@
         },
         "symfony-app-dir": "app",
         "symfony-web-dir": "web"
-    }
+    },
+    "minimum-stability": "dev"
```

Comment:
i don't think you should do this, you should require `"coduo/php-matcher": "2.1.*@dev",` explicitly